README.md

```diff
@@ -9,9 +9,9 @@ Creates the following resources:
 
 ## Terraform Versions
 
-Terraform 0.12. Pin module version to ~> 2.0.0. Submit pull-requests to master branch.
+Terraform 0.13 and later. Pin module version to ~> 3.0.0. Submit pull-requests to master branch.
 
-Terraform 0.11. Pin module version to ~> 1.0.0. Submit pull-requests to terraform011 branch.
+Terraform 0.12. Pin module version to ~> 2.0.0. Submit pull-requests to terraform011 branch.
```

Comment:
Do we want a terraform012 branch here instead?

Reply:
IDK how I missed that but you are right.